druzhba/postgres.py

```diff
@@ -123,24 +145,31 @@ def get_sql_description(self, sql):
             ) as cursor:
                 cursor.execute(sql.rstrip("; \n") + " LIMIT 1")
                 return (
+                    table_attributes,
                     (
-                        col.name,
-                        self.type_map.get(
-                            self.pg_types[col.type_code], self.pg_types[col.type_code],
-                        ),
-                        None,
-                        col.internal_size,
-                        col.precision,
-                        col.scale,
-                        column_is_nullable.get(col.name),
-                    )
-                    for col in cursor.description
+                        (
+                            col.name,
+                            self.type_map.get(
```

Comment:
we could probably get all of these columns from `information_schema.columns`, except I'm not sure how to  get something equivalent to a `type_code` here from https://www.postgresql.org/docs/9.5/infoschema-columns.html

don't need for this PR anyway